addons/rating/static/src/new/messaging_menu.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.messaging_menu.content" t-inherit-mode="extension">
+        <xpath expr="//NotificationItem[@body='preview.body']" position="attributes">
+            <attribute name="rating">preview.mostRecentMsg?.rating</attribute>
```

Comment:
Shouldn't the same possibly also apply to `mostRecentNeedactionMsg`?

Reply:
The channel preview and thread needaction preview are same component and view in the new code unlike the old code. I am not sure If it is the point of comment although.

Reply:
I added `thread needaction preview` to this PR.

Reply:
Depending on the preview type it's not the same message that is displayed, needaction does not display the `mostRecentMsg`, it displays the `mostRecentNeedactionMsg`.

Reply:
Follow up PR for proper use of  `mostRecentMsg` and `mostRecentNeedactionMsg`.